### ISSUE TITLE
GitHub Actions Workflow badge link does not provide ALT text

### ISSUE THREAD

**Issue by atc0005** [atc0005/elbow#145]
Error from linter:

```
MD045/no-alt-text: Images should have alternate text (alt text)markdownlint(MD045)
```

See also #144.
